plugins/BEdita/Core/src/Model/Entity/ObjectTag.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\Core\Model\Entity;
+
+use BEdita\Core\Utility\JsonApiSerializable;
+use Cake\ORM\Entity;
+
+/**
+ * ObjectTag Entity
+ *
+ * @property int $id
+ * @property int $object_id
+ * @property int $tag_id
+ *
```

Comment:
minor, remove newline between `@property` and others phpdoc (for cake4 phpcs compatibility)